plugins/gradle.properties

```diff
@@ -0,0 +1,2 @@
+gson_version=2.7
```

Comment:
gson version i mms frontend current is 2.8.5, should we make it uniform?

Reply:
We can. this is a reference implementation :) . But sure, I can make